### ISSUE TITLE
Why not use model.train() in synthesizer's fit mode and model.eval() in synthesizer's sample mode?

### ISSUE THREAD

**Issue by Zhengxuru** [sdv-dev/SDGym#22]
Hi I am new to Pytorch. Some layers have different behavior during train/and evaluation (like BatchNorm, Dropout), but I can't see model's train() mode and eval() mode in the code of synthesizer's fit mode or sample mode. Why don't use train mode or eval mode?

**Comment by leix28** [comment]
Hi @Zhengxuru,
This is a bug introduced when refactoring the code. Eval mode should be used when sampling synthetic data. We will address it soon.
